src/Source.ml

```diff
@@ -92,9 +92,9 @@ let extend_loc_to_include_attributes t (loc : Location.t)
           let loc =
             match payload with
             | PStr [] -> loc
-            | PStr l -> (List.last_exn l).pstr_loc
+            | PStr l -> (List.last_exn l).Parsetree.pstr_loc
```

Comment:
This was just a warning.

Reply:
I think it's fine to have warning trigger with older version of ocaml, @jberdine, any opinion ?